client/src/pages/Episodes/Episodes.js

```diff
@@ -33,6 +34,24 @@ class Episodes extends Component {
         })).then(console.log(this.state)).catch(err => console.log(err));
   };
 
+  // Create a function that will post the selected podcast to the db
+  postPodcast = () => {
+    const selectedPodcast = {
+      feedUrl: this.props.location.state.feedUrl,
+      artistId : this.props.location.state.artistid,
+      collectionId : this.props.location.state.collectionid,
+      artistName : this.props.location.state.artistname,
+      collectionName : this.props.location.state.collectionname,
+      artworkUrl30 : this.props.location.state.artworkurl30,
+      artworkUrl60 : this.props.location.state.artworkurl60,
+      artworkUrl100 : this.props.location.state.artworkurl100,
+      mostRecentRelease : this.props.location.state.releasedate,
+      genre : this.props.location.state.genre,
+    }
+    //Add the podcast to the API
+    API.addPodcast(selectedPodcast)
+  }
```

Comment:
This seems like it would be a better fit in the podcast card as a event handler onSubmit of podcast. For example:
```
handleSubmit(event) {
  API.addPodcast
}
```
Also, is this posting episode or podcast?

Reply:
Yes it would be...the problem was that I could not get the props to pass as it was pulling in the props for all the podcast cards instead of the one that was clicked. I was talking to Sam about this and he recommended doing this. I definitely want to refactor this if we get the chance but not sure how to get the props working correctly.

This is just the podcast show for now...episode will be next. Had to ponder how to pass the props for a played podcast. I think I have an idea @mbychkowski